### ISSUE TITLE
[Accessibility] Dropdown should be defined as 'read-only combobox' instead of 'editable combobox'

### ISSUE THREAD

**Issue by satbai** [OfficeDev/office-ui-fabric-react#4262]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No
Requested priority: Normal
Products/sites affected: (if applicable)

#### Describe the issue:

1. Go to https://developer.microsoft.com/en-us/fabric#/components/dropdown
2. Set focus to Basic uncontrolled example DropDown
3. Narrator announces 'Home - Office UI Fabric - Microsoft Edge, Basic dropdown example, editable combo box,'

![image](https://user-images.githubusercontent.com/24945574/37372309-8825195e-26cf-11e8-9319-0f6a84fbf6ca.png)

#### Expected behavior:

3. It should be a read only combo box. Shouldn't editable combo box have an actual input field that you can type into.

**Comment by michaelangotti** [comment]
@betrue-final-final What are your thoughts?

**Comment by betrue-final-final** [comment]
I agree. I meant to comment earlier that it should not be read as editable combo box. @cschlechty on whether 'read-only' means can't type values or can't use. I think this is correct but would love the double verification.

**Comment by cschlechty** [comment]
aria-readonly means you can't change the value of the control. It looks correct as-is https://www.w3.org/WAI/PF/aria/states_and_properties#aria-readonly

**Comment by betrue-final-final** [comment]
Ok, and "editable" specifically means it can be modified, and not just text editable?

**Comment by cschlechty** [comment]
yes, i believe so

**Comment by michaelangotti** [comment]
@satbai According to design this is expected. Closing this issue.
